test.sh

```diff
@@ -1,35 +0,0 @@
-#! /bin/bash
```

Comment:
The reason of removing this file is? Or to ask do we possibly know the original purpose of adding this file?

Reply:
It's used to test the fixture and integrate CI. But we have used our ruby framework to integrate CI. I have left it not used at last release. But it seems it causes confusion for someone read the code. So I decided to delete it.